elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -51,30 +53,52 @@
     private static HealthCheckRegistry healthCheckRegistry = null;
 
     /**
-     * Constructor
+     * Constructor.
      *
-     * @param injector Injection instance for application
+     * @param injector Injection instance for application.
+     * @param servletContext servlet context instance.
      */
     @Inject
     public ElideResourceConfig(ServiceLocator injector, @Context ServletContext servletContext) {
         this.injector = injector;
 
         settings = (ElideStandaloneSettings) servletContext.getAttribute(ELIDE_STANDALONE_SETTINGS_ATTR);
 
+        if (settings.enableDynamicModelConfig()) {
+            try {
+                Util.initDynamicConfig(settings.getDynamicConfigPath());
+            } catch (Exception e) {
```

Comment:
What exception are we catching here and why.  We need to be very careful about catching exception - in general - not advised.

Reply:
The DynamicEntityompiler compile method throws an Exception so this was catching and throwing that. Have updated DynamicCompiler to catch it and throw a run time exception now instead of compile time.